pkg/apis/servicecatalog/types.go

```diff
@@ -392,6 +393,10 @@ type ServiceInstanceStatus struct {
 	// the service instance.
 	DashboardURL *string
 
+	// CurrentOperation is the operation the Controller is currently performing
+	// on the ServiceInstance.
+	CurrentOperation ServiceInstanceOperation
```

Comment:
This should be a pointer, because it should be nil when there's no operation in progress.